<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PC-060" ignored="0" name="Gradle test" tests="5" failures="1" timestamp="10 окт 2018 08:53:08 GMT" time="3.365" errors="0">
  <testcase name="testCreateUser" time="2.537" classname="one.protocol.api.tests.APITests"/>
  <testcase name="testCreateUser" time="0.167" classname="one.protocol.api.tests.APITests"/>
  <testcase name="testCreateUser" time="0.166" classname="one.protocol.api.tests.APITests"/>
  <testcase name="testCreateUser" time="0.175" classname="one.protocol.api.tests.APITests"/>
  <testcase name="testSocNets" time="0.218" classname="one.protocol.api.tests.APITests">
    <failure type="java.lang.AssertionError" message="expected [[{&amp;quot;name&amp;quot;:&amp;quot;facebook&amp;quot;,&amp;quot;url&amp;quot;:&amp;quot;\/oauth\/connect\/facebook&amp;quot;},{&amp;quot;name&amp;quot;:&amp;quot;vkontakte&amp;quot;,&amp;quot;url&amp;quot;:&amp;quot;\/oauth\/connect\/vkontakte&amp;quot;},{&amp;quot;name&amp;quot;:&amp;quot;odnoklassniki&amp;quot;,&amp;quot;url&amp;quot;:&amp;quot;\/oauth\/connect\/odnoklassniki&amp;quot;}]] but found [[{&amp;quot;name&amp;quot;:&amp;quot;vkontakte&amp;quot;,&amp;quot;url&amp;quot;:&amp;quot;\/oauth\/connect\/vkontakte&amp;quot;}]]">
      <![CDATA[java.lang.AssertionError: expected [[{"name":"facebook","url":"\/oauth\/connect\/facebook"},{"name":"vkontakte","url":"\/oauth\/connect\/vkontakte"},{"name":"odnoklassniki","url":"\/oauth\/connect\/odnoklassniki"}]] but found [[{"name":"vkontakte","url":"\/oauth\/connect\/vkontakte"}]]
at one.protocol.api.tests.APITests.testSocNets(APITests.java:45)
at com.sun.proxy.$Proxy1.stop(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 52 stack frames]]>
    </failure>
  </testcase> <!-- testSocNets -->
</testsuite> <!-- Gradle test -->
